diagnostic.md

```diff
@@ -7,40 +7,43 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone https://github.com/conorjennings/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
+Change into the new directory from the clone command
+ cd git-diagnostic/
```

Comment:
Hey Conor this is close, we were looking for creating a new branch so that you don't do your work on the master branch. But this was a good start.